archiver/archiver.py

```diff
@@ -80,6 +80,9 @@ def __init__(self, ingest_api: IngestAPI, manifest_id: str):
         self.sequencing_protocol = None
         self.input_biomaterial = None
 
+    def get_submission_uuid(self):
+        return self.manifest.get('envelopeUuid')
```

Comment:
Is the manifest has always got the `envelopeUuid`?
or shall we replace to this line:
`return self.manifest.get('envelopeUuid', None)`

Reply:
The get method returns None by default if the key is missing.